### PR TITLE
Add terminal connection state for battery in tab view

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/dto/definition/hvdc/HvdcTabInfos.java [hunk 1]
```diff
@@ -54,6 +54,4 @@ public class HvdcTabInfos extends ElementInfos {
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private Float oprFromCS2toCS1;
-
-
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/network/map/dto/definition/hvdc/HvdcTabInfos.java:58]: you should revert this file

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: ok, but revert HvdcTabInfos.java
tests: ok

File: src/main/java/org/gridsuite/network/map/dto/definition/hvdc/HvdcTabInfos.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/network/map/dto/definition/hvdc/HvdcTabInfos.java]: Done

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Review by antoinebhs — APPROVED** [review]
Code OK
Tests OK

**Pull request merged**